routes/api.php

```diff
@@ -2,17 +2,36 @@
 
 use Illuminate\Http\Request;
 
-/*
-|--------------------------------------------------------------------------
-| API Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register API routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| is assigned the "api" middleware group. Enjoy building your API!
-|
-*/
-
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::group(['middleware' => 'auth.jwt'], function () {
+    Route::post('register', 'AuthController@register');
+    Route::post('login', 'AuthController@login');
+    Route::get('logout', 'AuthController@logout');
+    Route::post('refresh', 'AuthController@refresh');
+    Route::post('me', 'AuthController@me');
+
+});
+
+
+Route::post('refresh', 'DataCollectorController@refresh');
+Route::post('me', 'DataCollectorController@me');
+Route::post('login', 'DataCollectorController@login');
+Route::get('user', 'DataCollectorController@getAuthUser');
+Route::get('logout', 'DataCollectorController@logout');
+Route::post('/register','DataCollectorController@register');
+Route::get('/collectors','DataCollectorController@index');
```

Comment:
@ssensalo These routes are still not protected by the middleware. This means that even an inauthentic user will directly access these sensitive routes

Reply:
Okay @Sebuliba-Adrian am going to fix that in the next commit.